media-libs/libpgf/libpgf-7.19.3.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Library to load, handle and manipulate images in the PGF format"
+HOMEPAGE="https://www.libpgf.org/"
+SRC_URI="https://master.dl.sourceforge.net/project/libpgf/libpgf/${PV}-latest/libPGF-codec-and-console-src.zip -> ${P}.zip"
```

Comment:
Is this the right URL to be using? It looks a bit weird (direct, not mirrored or something?)

Reply:
Hmm, I picked this up from tarball downloaded at sourceforge, it was probably using different mirrors.\
I see that you updated SRC_URI already, so thanks for that!